src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -559,14 +569,24 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
 
     if (!!showValue) {
       if (this.state.useRegex) {
-        filteredLogs = filteredLogs.filter(line => line.match(showValue));
+        try {
+          const regexp = new RegExp(showValue);
+          filteredLogs = filteredLogs.filter(l => regexp.test(l));
+        } catch (e) {
```

Comment:
Wow, I didn't know the filtering could fail

Reply:
the filtering won't likely fail but the constructor can fail to parse the regex string, so this way we only filter good regex.